plugins/Transitions/javascripts/transitions.js

```diff
@@ -249,18 +249,22 @@ Piwik_Transitions.prototype.preparePopover = function () {
     if (self.actionType == 'url') {
         title = Piwik_Transitions_Util.shortenUrl(title, true);
     }
-    title = self.centerBox.find('h2')
-        .addClass('Transitions_ApplyTextAndTruncate')
-        .data('text', title)
-        .data('maxLines', 3);
-
+    var h2 = self.centerBox.find('h2');
```

Comment:
Can you maybe explain these changes?

Reply:
Instead of using `window.open()`, links are created using regular <a> tags, so they use the same mechanism for handling `noreferrer noopener` and `action=redirect&url=...` as everywhere else.

This allows right-click on links and selecting “Copy URL” from the context menu etc.

It was relevant in my first version of this PR before I reverted to use `action=redirect&url=...` (due to IE 10). I still think it is a good change (it streamlines `noreferrer` handling and avoids the UI problems with `window.open()`), but I guess it is not directly related to this PR. So I'll revert the changes from the PR and perhaps create a separate PR instead.